README.md

```diff
@@ -169,6 +171,27 @@ You can import word templates by adding `<documenttemplate>` elements.
 </templateconfig>
 ```
 
+### Attribute specific functionality
+
+#### Set auto-number seed values
+
+When deploying auto-numbers, seed values can be defined in the template for each entity. These are set post-deployment. Setting the `<autonumberseedvalue>` element determines that the column is an auto-number.
+
+```xml
+<templateconfig>
+    <tables>
+      <table name="account">
+        <columns>
+          <column name="new_accountautonumber" autonumberseedvalue="1"/>
+        </columns>
+      </table>
+      <table name="contact">
+        <columns>
+          <column name="new_contactautonumber" autonumberseedvalue="2000"/>
+        </columns>
+</templateconfig>
```

Comment:
Missing the closing `</tables>` tag. Also in the interest of conciseness, perhaps just one of these should do.

```suggestion
<templateconfig>
    <tables>
      <table name="account">
        <columns>
          <column name="new_accountautonumber" autonumberseedvalue="1"/>
        </columns>
      </table>
   </tables>
</templateconfig>
```

Reply:
Wouldn't keeping an example of two tables be better? It would make it obvious that you can set multiple seeds across entities and how that should look.

Will fix the tags.